corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -189,7 +189,10 @@ def get_data_schema(request, domain, app_id=None, form_unique_id=None):
     if app.domain != domain:
         raise Http404()
     data.append(get_casedb_schema(app))  # TODO use domain instead of app
-    data.extend(item_lists_by_domain(domain))
+    data.extend(
+        sorted(item_lists_by_domain(domain),
+               cmp=lambda x, y: cmp(x['name'].lower(), y['name'].lower()))
```

Comment:
You're right, not deprecated. It's just ["the old way."](https://wiki.python.org/moin/HowTo/Sorting#The_Old_Way_Using_the_cmp_Parameter)